### DISCUSSION TITLE
I'm getting a fatal error when running an action

### DISCUSSION THREAD

**Discussion by djokhar05** [community/community#136640] [Actions] [answered]
### Select Topic Area

Question

### Body

Hi, i'm getting this error when this workflow:

name: (main) push to external repo
on:
  push:
    branches:
      - main
jobs:
  push-to-external-repo:
    runs-on: ubuntu-latest
    steps:
      - name: checkout repository
        uses: actions/checkout@v3
      - name: push to external repository
        uses: peaceiris/actions-gh-pages@v3
        with:
          deploy_key: ${{ secrets.SSH_DEPLOY_KEY }}
          publish_dir: .
          external_repository: <djokhar05/bitteller-server-dest.git>
          publish_branch: main
          allow_empty_commit: true

**and this is the erorr i get**

/usr/bin/git push origin main
fatal: remote error:
is not a valid repository name

could anyone help me on this please

**Comment by R-I-T-U** [comment, marked as answer]
The <djokhar05/bitteller-server-dest.git> placeholder seems incorrect. Ensure that the external_repository field is correctly formatted. It should look like this:
external_repository: djokhar05/bitteller-server-dest

**Reply by djokhar05** [reply]
oh wow! thanks a lot, i'll try it right away

**Reply by djokhar05** [reply]
thank you so much it worked fine after the change

**Reply by R-I-T-U** [reply]
Glad to know... Happy coding!

**Reply by HACKER007-SHIP-IT** [reply]
This was mine

**Comment by alidolem06** [comment]
freeopen

**Comment by HACKER007-SHIP-IT** [comment]
First steel then delete my repo

**Comment by w17929448-watcharin** [comment]
<details><summary>Details</summary>
<p>



</p>
</details>
